pkg/deploy/api/types.go

```diff
@@ -437,6 +437,8 @@ type DeploymentCondition struct {
 	Type DeploymentConditionType
 	// Status of the condition, one of True, False, Unknown.
 	Status kapi.ConditionStatus
+	// The last time this condition was updated.
+	LastUpdateTime unversioned.Time
```

Comment:
@kargakis what is the difference between `LastTransitionTime` and `LastUpdateTime` ?